specifyweb/frontend/js_src/lib/components/WbPlanView/CustomSelectElement.tsx

```diff
@@ -573,9 +573,9 @@ export function CustomSelectElement({
         aria-expanded={isOpen}
         aria-haspopup="listbox"
         className={`
-          flex min-h-[theme(spacing.8)] min-w-max
-          cursor-pointer items-center gap-1 rounded border 
-          border-gray-500 px-1 dark:border-none md:min-w-[unset]
+          flex min-h-[theme(spacing.8)] min-w-max cursor-pointer
+          items-center gap-1 rounded border border-gray-500
+          px-1 text-left dark:border-none md:min-w-[unset]
```

Comment:
can you please check why this bug wasn't an issue before, but is an issue now on this branch?
was something else changed that caused this to happen? or was some code removed accidentally (or maybe in a process of resolving merge conflict?)

just want to double check that whatever caused this to not work, didn't break anything else alongside. thanks
(it could have been just a user error on my side, but hoping it's not anything more than that)

Reply:
In Edge: 
```
preview = (
      // Not tabbable because keyboard events are handled separately
      // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/interactive-supports-focus
      <header
        aria-expanded={isOpen}
        aria-haspopup="listbox"
        className={`
          flex min-h-[theme(spacing.8)] min-w-max
          cursor-pointer items-center gap-1 rounded border 
          border-gray-500 px-1 dark:border-none md:min-w-[unset]
          ${
            defaultOption?.isRequired === true
              ? 'custom-select-input-required bg-[color:var(--custom-select-b2)]'
              : defaultOption?.isHidden === true
              ? `custom-select-input-hidden bg-[color:var(--custom-select-b2)]
                dark:!border-solid`
              : customSelectType === 'OPTIONS_LIST' &&
                defaultOption?.isRelationship === true
              ? 'bg-yellow-250 dark:bg-yellow-900'
              : customSelectElementBackground
          }
          ${isOpen ? 'z-[3] rounded-b-none' : ''}
        `}
        role="button"
        onClick={
          has('interactive') ? (isOpen ? handleClose : handleOpen) : undefined
        }
      >
```
      
      In XML: 
      ` preview = (
      // Not tabbable because keyboard events are handled separately
      <button
        aria-controls={id('options')}
        aria-describedby={id('validation')}
        aria-expanded={isOpen}
        aria-haspopup="listbox"
        className={`
          flex min-h-[theme(spacing.8)] min-w-max cursor-pointer
          items-center gap-1 rounded border border-gray-500
          px-1 text-left dark:border-none md:min-w-[unset]
          ${
            defaultOption?.isRequired === true
              ? 'custom-select-input-required bg-[color:var(--custom-select-b2)]'
              : defaultOption?.isHidden === true
              ? `custom-select-input-hidden bg-[color:var(--custom-select-b2)]
                dark:!border-solid`
              : customSelectType === 'OPTIONS_LIST' &&
                defaultOption?.isRelationship === true
              ? 'bg-yellow-250 dark:bg-yellow-900'
              : customSelectElementBackground
          }
          ${isOpen ? 'z-[3] rounded-b-none' : ''}
        `}
        disabled={handleClick === undefined}
        type="button"
        onClick={handleClick}
      >`

Reply:
Tested: 
Changing <button> by <header> align text to the left without the need to add text-left in className